distributed/settings.yaml

```diff
@@ -0,0 +1,88 @@
+jobs:
```

Comment:
@JoanFM @cristianmtr 
Instead of running the stress-test in bash scripts, we can define a yaml file like this.

Reply:
good!